src/sentry/web/frontend/organization_integration_setup.py

```diff
@@ -20,46 +20,34 @@ class OrganizationIntegrationSetupView(ControlSiloOrganizationView):
     csrf_protect = False
 
     def handle(self, request: Request, organization, provider_id) -> HttpResponseBase:
-        try:
-            with sentry_sdk.configure_scope() as scope:
-                parent_span_id = scope.span.span_id
-                trace_id = scope.span.trace_id
-        except AttributeError:
-            parent_span_id = None
-            trace_id = None
+        with sentry_sdk.configure_scope() as scope:
+            scope.transaction.op = "integration.setup"
```

Comment:
is it important to be setting the `op` here as well?

Reply:
no, the op will just be the normal django http request stuff from the integration.